packages/itinerary-body/src/AccessLegBody/index.tsx

```diff
@@ -104,6 +104,7 @@ class AccessLegBody extends Component<Props, State> {
             showLegIcon={showLegIcon}
           />
           <S.StepsHeader
+            role="button"
```

Comment:
Is this needed? It is already a `<button>`.

Reply:
Do we need an `aria-controls` or `aria-owns` attribute?

Reply:
Yes the `role="button"` seems to not be needed, but `aria-expanded` is what seems to be needed.